pkgs/servers/nextcloud/packages/default.nix

```diff
@@ -2,7 +2,13 @@
 # Licensed under: MIT
 # Slightly modified
 
-{ lib, pkgs, newScope, apps }:
+{
+  lib
+, pkgs
+, newScope
+, apps
+, callPackage
+}:
```

Comment:
The formatting looks weird. If we format stuff, we should use the rfc-compliant formatter.